packages/shared/useDateFormat/index.md

```diff
@@ -27,11 +27,17 @@ Get the formatted date according to the string of tokens passed in, inspired by
 | `s`    | 0-59             | The second                            |
 | `ss`   | 00-59            | The second, 2-digits                  |
 | `SSS`  | 000-999          | The millisecond, 3-digits             |
+| `A`    | AM PM            | The meridiem                          |
+| `AA`   | A.M. P.M.        | The meridiem, periods                 |
+| `a`    | am pm            | The meridiem, lowercase               |
+| `aa`   | a.m. p.m.        | The meridiem, lowercase and periods   |
 | `d`    | 0-6              | The day of the week, with Sunday as 0 |
 | `dd`   | S-S              | The min name of the day of the week   |
 | `ddd`  | Sun-Sat          | The short name of the day of the week |
 | `dddd` | Sunday-Saturday  | The name of the day of the week       |
 
+- Meridiem is customizable by defining `customMeridiem` in `options`.
```

Comment:
How about we add a usage example for it ?

Reply:
I've added the usage guide for it. Thanks for the suggestion